src/lib/mongoBackend/MongoGlobal.cpp

```diff
@@ -80,6 +80,7 @@ static std::string          subscribeContextCollectionName;
 static std::string          subscribeContextAvailabilityCollectionName;
 static std::string          assocationsCollectionName;
 static Notifier*            notifier;
+static bool                 multitenant;
```

Comment:
We already have a global variable 'broker wide', called 'mtenant'. Why not use it?